Source/ZoomNet/Resources/IToken.cs

```diff
@@ -0,0 +1,19 @@
+using System.Threading;
+using System.Threading.Tasks;
+using ZoomNet.Models;
+
+namespace ZoomNet.Resources
+{
+	/// <summary>
+	/// Allows you to manage tokens.
+	/// </summary>
+	public interface IToken
+	{
+		/// <summary>
+		/// To revoke a token.
+		/// </summary>
+		/// <param name="cancellationToken">Cancellation token.</param>
+		/// <returns>TokenRevocation object.</returns>
+		Task<TokenRevocation> RevokeAsync(CancellationToken cancellationToken = default);
```

Comment:
Do you think we can simplify it and just move this method to the `ZoomOAuthClient ` class?

Reply:
`Token` resource has been removed and the `RevokeAsnyc` method moved inside the client